cli/src/semgrep/core_output.py

```diff
@@ -27,6 +28,14 @@
 logger = getLogger(__name__)
 
 
+def _core_location_to_error_span(location: core.Location) -> out.ErrorSpan:
```

Comment:
Should core location export this functionality?

Reply:
Or ErrorSpan expose some classmethod instantiator that takes a core.Location object?

Reply:
core.* is generated by ATD, so I don't think there's a good way to edit it.

I'm sure we could get fancy with ErrorSpan, but since this code was only used within this file it made sense just to crowbar it into a private function.

Reply:
sg we can move if ever gets used more